.envrc

```diff
@@ -0,0 +1,5 @@
+# Set GOTOOLCHAIN to local to prevent automatic Go version updates
```

Comment:
This could be different for different people's environments, right? Do we want to keep this in source?

Reply:
I think this got slipped through, let me remove it